src/main/java/appeng/client/render/cablebus/CableBusBakedModel.java

```diff
@@ -176,7 +179,7 @@ private void addCableQuads( CableBusRenderState renderState, List<BakedQuad> qua
 
 		// If the connection is straight, no busses are attached, and no covered core has been forced (in case of glass
 		// cables), then render the cable as a simplified straight line.
-		boolean noAttachments = renderState.getAttachments().isEmpty();
+		boolean noAttachments = !renderState.getAttachments().entrySet().stream().filter( p -> p.getValue().isSolid() ).findAny().isPresent();
 		if( isStraightLine( cableType, connectionTypes ) && noAttachments )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method appeng.client.render.cablebus.CableBusBakedModel.addCableQuads(CableBusRenderState, List) orders expressions in a conditional in a sub optimal way [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=fb-contrib%3ASEO_SUBOPTIMAL_EXPRESSION_ORDER)